alembic/versions/2024_05_13_078884a6b2d2_fix_new_tech_tables.py

```diff
@@ -0,0 +1,29 @@
+"""fix_new_tech_tables
+
+Revision ID: 078884a6b2d2
+Revises: 9b188aee9577
+Create Date: 2024-05-13 11:36:49.628610
+
+"""
+
+import sqlalchemy as sa
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "078884a6b2d2"
+down_revision = "9b188aee9577"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.drop_column("run_device", "device_id")
+    op.add_column("run_metrics", sa.Column("device_id", sa.Integer(), nullable=False))
+    op.create_foreign_key("fk_device_id", "run_metrics", "run_device", ["device_id"], ["id"])
```

Comment:
I think I didnt create foreign keys ont he other tables either, should we add the samplerun etrics keys as well?=

Reply:
Feel free to add it in a separate PR